com.github.jean28518.Linux-Assistant.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.github.jean28518.Linux-Assistant
```

Comment:
this must be `io.github`.

Reply:
Should I open a new pr or is there a possibility to rename my branch?

Reply:
the branch name is irrelevant